src/jit/lower.cpp

```diff
@@ -2862,8 +2862,8 @@ void Lowering::InsertPInvokeMethodProlog()
     firstBlockRange.InsertBefore(insertionPoint, LIR::SeqTree(comp, store));
     DISPTREERANGE(firstBlockRange, store);
 
-#ifndef _TARGET_X86_ // For x86, this step is done at the call site (due to stack pointer not being static in the
-                     // function).
+#if !defined(_TARGET_X86_) && !defined(_TARGET_ARM_)
+    // For x86 and arm32, this step is done at the call site (due to stack pointer not being static in the function).
 
```

Comment:
Any of `DomainNeutralILStubClass:IL_STUB_PInvoke`. Legacy output is here on github in my neighbour comment; full RyuJIT's one I'll be able to show only tomorrow...

It seems that the approach to make ryujit output identical to the legacy one is not the best for every case.